From be5da0c753cfcec84b397e7a63fbd1db2c2eac90 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi2@arcor.de>
Date: Wed, 5 Apr 2017 18:28:19 +0200
Subject: [PATCH 10/74] mirror: Add some nvidia repositories

 - Added: android_hardware_nvidia_libstagefrighthw
 - Added: android_hardware_nvidia_thermal

Change-Id: I870b12736d84875f48462cd5af2f457b319277bb
---
 default.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/default.xml b/default.xml
index 232c69b..1381a19 100644
--- a/default.xml
+++ b/default.xml
@@ -1172,8 +1172,10 @@
   <project name="LineageOS/android_hardware_libhardware_legacy" />
   <project name="LineageOS/android_hardware_msm7k" />
   <project name="LineageOS/android_hardware_nvidia_audio" />
+  <project name="LineageOS/android_hardware_nvidia_libstagefrighthw" />
   <project name="LineageOS/android_hardware_nvidia_power" />
   <project name="LineageOS/android_hardware_nvidia_tegra124" />
+  <project name="LineageOS/android_hardware_nvidia_thermal" />
   <project name="LineageOS/android_hardware_qcom_audio" />
   <project name="LineageOS/android_hardware_qcom_audio-bfam" />
   <project name="LineageOS/android_hardware_qcom_audio-caf" />
-- 
2.17.1

